seed/seedwriter/seed20.go

```diff
@@ -343,8 +345,27 @@ func (tr *tree20) writeAssertions(db asserts.RODatabase, modelRefs []*asserts.Re
 	}
 
 	if len(extraRefs) != 0 {
-		if err := writeByRefs("extra-assertions", refsGen(extraRefs, excludeModel)); err != nil {
-			return err
+		// system-user assertions must be placed in a file called auto-import.assert to be auto loaded
+		var (
+			foundSystemUserAssert bool
+			foundExtraAssert      bool
+		)
```

Comment:
This is more in line with our current coding style
```suggestion
		var foundSystemUserAssert, foundExtraAssert bool
```